index.html

```diff
@@ -200,6 +200,38 @@ <h2 class="interest">I'm always interested in hearing about new projects, so if
             </form>  
             </div>    
         </section>
+
+        <section class="pop-up">
+            <div id="popup-head">
+            <h2 id="pop-up-heading">Title</h2>
+            <button id="cancel-pup-up" type="submit"><img src="images/cancel.png" alt=""></button>
+            </div>
+            <ul id="pop-up-tech">
+                <li id ="tech-skill1"></li>
+                <li id ="tech-skill2"></li>
+                <li id ="tech-skill3"></li>
+            </ul>
+            <div class="popup-container1">
+            <img src="would-be-replaced" alt="" id="featured-image">
+            <img src="would-be-replaced" alt="" id="featured-image-big">
+            <div class="popup-container2">
+            <p id="pop-up-description" ></p>
+            <div id="pop-up-foot">
+                <div id="live">
+                    <h2 id="live-text" class="down-text">See Live</h2>
+                    <img id="live-icon" src="images/Icon1.png" alt="source icon">
+                </div>
+                <div id="source">
+                    <h2 id="source-text" class="down-text">See Source</h2>
+                    <img id="source-icon" src="images/Vector1.png" alt="Github icon">
+                </div>
+            </div>
+            </div>
+            </div>
+        </section>
+        <div id="overlay">
+        </div>
```

Comment:
- [ ] According to the requirements Modal popup data is populated with the data from the projects array as shown here so kindly add it as array of objects and dynamically by the js file


![image](https://user-images.githubusercontent.com/58553711/180353486-38a24fb0-c34d-4548-aab4-27c846b487d1.png)

Reply:
@Reem-lab Thank you for your review. But can you please elaborate more on what changes I need to make? I'm a little confused

Reply:
ok, sure Let me explain more I think you should put all the information in array of objects and then you can create the pop up inside your javascript file and create the pop dynamically by something called `innerHtml` you can check this link [how to create html inside javascript file](https://www.geeksforgeeks.org/how-to-append-html-code-to-a-div-using-javascript/)


![image](https://user-images.githubusercontent.com/58553711/180446843-3305e3fe-775c-4612-9049-1d7a731f5a7f.png)

Reply:
Okay, @Reem-lab. Thanks a lot for the explanation. It was helpful. I've made all the necessary changes.